11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        def pred(self, feature_x, a_vec):
</pre><p>Need to add type annotation:</p><pre class="ql-syntax" spellcheck="false">feature_x: torch.Tensor, a_vec: torch.Tensor
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>